nixos/modules/services/networking/dsnet.nix

```
@@ -0,0 +1,360 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.dsnet;
+  settingsFormat = pkgs.formats.json { };
+  patchFile = settingsFormat.generate "dsnet-patch.json" cfg.settings;
+in
+{
+  options.services.dsnet = {
+    enable = lib.mkEnableOption "dsnet, a centralised Wireguard VPN manager";
+
+    package = lib.mkOption {
+      type = lib.types.package;
+      default = (lib.mkPackageOption pkgs "dsnet" { });
+      description = "The dsnet package to use.";
+    };
```

Nono, not like this.

```suggestion
    package = lib.mkPackageOption pkgs "dsnet" { };
```